apis/github

```diff
@@ -71,10 +79,13 @@ end
 local Tree = {}
 Tree.__index = Tree
 Tree.new = function(repo, sha, path)
-	local data = getAPI(
+	local status, data = getAPI(
 		('repos/%s/%s/git/trees/%s'):format(repo.user, repo.name, sha),
 		repo.auth
 	)
+	if not status then
+		error('Could not get github API from ' ..('repos/%s/%s/git/trees/%s'):format(repo.user, repo.name, sha))
```

Comment:
should we create the string first and then pass it to error?
i.e.
```
local pathStr = ('repos/%s/%s/git/trees/%s'):format(repo.user, repo.name, sha)
local errStr = ('Could not get github API from %s'):format(pathStr)
error(errStr)
```

edit: this is actually a syntax error... the question still stands though.